libp2p/protocols/kademlia/types.nim

```diff
@@ -250,6 +252,27 @@ method isValid*(
 ): bool {.raises: [], gcsafe.} =
   return true
 
+type LogosEntryValidator* = ref object of EntryValidator
```

Comment:
This too seems very app specific and should likely not be part of libp2p kad-dht